src/views/partials/podcasts/create.hbs

```diff
@@ -0,0 +1,27 @@
+<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
+<script src="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.js"></script>
+
+<form action="/podcasts/create" method="post" autocomplete="off" enctype="application/json">
+  <div class="form-group col-md-6">
+    <label>{{{__ "videopost_title"}}}:</label>
+    <input type="text" class="form-control" name="title" required>
+  </div>
+  <div class="form-group col-md-6">
+    <label>{{{__ "videopost_author"}}}:</label>
+    <input type="text" class="form-control" name="author" required>
+  </div>
+  <div class="form-group col-md-6">
+    <label>{{{__ "videopost_id"}}}:</label>
+    <input type="text" class="form-control" name="video_id" required>
+  </div>
+  <div class="col-md-6">
+    <input type="submit" value="{{{__ "common_submit"}}}">
+    <input type="reset">
+  </div>
+</form>
+
+<script>
+  var simplemde = new SimpleMDE({
+    element: document.getElementById("main-content")
+  });
+</script>
```

Comment:
Do we need this or was it copy-pasted from the blog and forgotten?

Reply:
Forgotten 😅